### PR TITLE
Aligned to new design

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 1]
```diff
@@ -134,28 +132,30 @@
...
+  font-weight: 400;
+  padding-top: 21px;
 }
 
 .button-login{
-  background-color: black;
+  background-color: #EA241A;
   color: white;
-  border: 1px red solid;
+  border: 1px white solid;
```
Comment (CilliersN) [inline skietbaan_fe/src/components/RegisterStyles.css]: above code is hex. change it to be the same

File: skietbaan_fe/src/components/RegisterStyles.css [hunk 2]
```diff
@@ -177,20 +177,39 @@
...
   margin-top: -4px;
 }
+
+.hidden{
+  display: none;
+}
+
+.header-image{
+  height: 94px;
+  width: 100%;
```
Comment (CilliersN) [inline skietbaan_fe/src/components/RegisterStyles.css]: Read this:
https://stackoverflow.com/questions/17468733/difference-between-width-auto-and-width-100-percent

File: skietbaan_fe/src/components/RegisterStyles.css (hunk 1 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/RegisterStyles.css]: alright fixed

File: skietbaan_fe/src/components/RegisterStyles.css (hunk 2 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/RegisterStyles.css]: Thanks read it but auto is not what I need to use in this case

Comment (CilliersN) [reply skietbaan_fe/src/components/RegisterStyles.css]: nope, was just a note

**Pull request merged**